styles.css

```diff
@@ -75,3 +85,261 @@ body {
 .fab {
   color: #dbd8d7;
 }
+
+/* Recent Works Section */
+
+.recent-works {
+  font-family: Crete Round, sans-serif;
+  font-style: normal;
+  font-weight: lighter;
+  font-size: 30px;
+  line-height: 52px;
+  text-align: center;
+  padding-top: 100px;
+  color: #1c1a19;
+  padding-bottom: 120px;
+}
+
+hr {
+  width: 330px;
+  border-color: black;
+}
+
+.recent-works img {
+  padding-top: 70px;
+  width: 327px;
+  height: 237px;
+}
+
+.recent-works h4 {
+  text-align: left;
+  font-family: Crete Round, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.recent-works p {
+  text-align: left;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.tech-used {
+  display: flex;
+  width: 250px;
+  justify-content: space-between;
+  padding: 0;
+}
+
+.recent-works li {
+  border: 1px solid black;
+  list-style: none;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  padding: 10px;
+  margin: 0 12px 0 0;
+}
+
+button {
+  background-color: #f55b00;
+  color: white;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+}
+
+button:hover {
+  background-color: #fff5e1;
+  border: 1px solid #f55b00;
+  color: #f55b00;
+}
+
+button:active {
+  background-color: #e05304;
+  color: #f4f1f0;
+}
+
```

Comment:
- [ ]  Nice job for making this button look the same as in the Figma design. You can make it look even super by adding the **Active**, **Hover**, **Pressed** button styling effects. buttons are a vital element in creating a positive and productive user experience.

Reply:
Hello @Hamzaoutdoors!
I'm sorry. I didn't understand what exactly are the requested changes here.
As you can see on the conde I've already added the :hover and :active selectors. And when I hover the mouse through the buttons, they change the background-color, border, and color. The same when I press them, they are the same as the Figma template.

Reply:
Hello @franciscoPonceDev
 - here I am just **highlighting** your efforts working on that 👍🏼 👍🏼, sorry for making you confused in this part

Reply:
Oh Thank you!! It is great to receive positive feedback from you! :)